lib/app/reload-actions.js

```diff
@@ -32,8 +32,8 @@ module.exports = function reloadActions(options, cb) {
   // Default `hooksToSkip` to an empty array.
   var hooksToSkip = options.hooksToSkip || [];
 
-  // The list of hooks we want to reload is the list of all hooks minus the hooks to skip.
-  var hooksToReload = _.difference(_.keys(sails.hooks), hooksToSkip);
+  // The list of activated hooks we want to reload is the list of all hooks minus the hooks to skip.
+  var hooksToReload = _.difference(_.filter(_.keys(sails.hooks), function(hook) { return sails.hooks[hook]; }), hooksToSkip);
```

Comment:
@atiertant Thanks!  This LGTM with a minor tweak:  wouldn't we want to change the comment as follows?

> The list of ~~activated~~ hooks we want to reload is the list of all **activated** hooks, minus the hooks to skip.

@athieriot Also, if you have a moment to add a test for this, I'd appreciate it.


@sgress454 would you mind taking a look at this to confirm, since you were in here last?

Thanks guys!